src/modules/LanItems.rb

```diff
@@ -2176,7 +2191,10 @@ def Commit
     # @return [true] so that this can be used for the :abort callback
     def Rollback
       log.info "rollback item #{@current}"
-      if getCurrentItem.fetch("hwinfo", {}).empty?
+      # Do not delete elements that are :edited but does not contain hwinfo
+      # yet (Add a virtual device and then edit it canceling the process during the
+      # edition)
+      if LanItems.operation == :add && getCurrentItem.fetch("hwinfo", {}).empty?
```

Comment:
Fix for some LanItem cancel editing issues:

- See https://bugzilla.suse.com/show_bug.cgi?id=998540